File: 백준/Bronze/5086. 배수와 약수/배수와 약수.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.StringTokenizer;

public class Main {
	
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		StringTokenizer st;
		
		while (true) {
			st = new StringTokenizer(br.readLine(), " ");
			
			int a = Integer.parseInt(st.nextToken());
			int b = Integer.parseInt(st.nextToken());
			
			if (a == 0 && b == 0)	break;
			
			if (b % a == 0) {
				System.out.println("factor");
			} else if (a % b == 0) {
				System.out.println("multiple");
			} else {
				System.out.println("neither");
			}
		}
		br.close();
	}
} 

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.cs
using System;

namespace Algorithm_Practice
{
    class Program
    {
        static void Main(string[] args)
        {
            while (true) {
                string[] str = Console.ReadLine().Split(" ");
                if (int.Parse(str[0]) == 0 && int.Parse(str[1]) == 0) {
                    break;
                }
                
                if (int.Parse(str[1]) % int.Parse(str[0]) == 0) {
                    Console.WriteLine("factor");
                } else if (int.Parse(str[0]) % int.Parse(str[1]) == 0) {
                    Console.WriteLine("multiple");
                } else {
                    Console.WriteLine("neither");
                }
            }
        }
    }
}

